src/components/include/test/policy/mock_policy_manager.h

```diff
@@ -82,48 +76,44 @@ class MockPolicyManager : public PolicyManager {
   MOCK_METHOD0(OnUpdateStarted, void());
   MOCK_CONST_METHOD1(GetUserConsentForDevice,
                      DeviceConsent(const std::string& device_id));
-  MOCK_METHOD3(
-      GetUserConsentForApp,
-      void(const std::string& device_id,
-           const std::string& policy_app_id,
-           std::vector<policy::FunctionalGroupPermission>& permissions));
+  MOCK_METHOD3(GetUserConsentForApp,
+               void(const std::string& device_id,
+                    const std::string& policy_app_id,
+                    std::vector<FunctionalGroupPermission>& permissions));
```

Comment:
@LevchenkoS , i think if it mock class, all includes must be included in parent header. Because if in parent header not included, it will be not build-able.